guides/doc-Quickstart_Guide/docinfo.xml

```diff
@@ -0,0 +1,15 @@
+<title>Installing Satellite Server - Connected</title>
```

Comment:
We only need this file for Satellite builds, so if this is Foreman-only, we can omit

Reply:
Thanks :) I was not sure so I just put everything I saw everywhere else in.